$Id$
$MirOS: ports/www/lynx/patches/patch-lynx_cfg,v 1.7 2007/05/20 12:06:19 tg Exp $

	mostly site policy and compatibility, usability

--- lynx2-8-7.orig/lynx.cfg	2007-05-18 00:53:22.000000000 +0200
+++ lynx2-8-7/lynx.cfg	2009-06-13 11:46:29.000000000 +0200
@@ -93,12 +93,12 @@
 #
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
-STARTFILE:http://lynx.isc.org/
+#STARTFILE:http://lynx.isc.org/
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
 .ex
-#STARTFILE:file://localhost/~/
+STARTFILE:file://localhost/~/
 #
 # Your choice of STARTFILE should reflect your site's needs, and be a URL that
 # you can connect to reliably.  Otherwise users will become confused and think
@@ -409,7 +409,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # values and their relationship to the locale value is not.  GNU libiconv
 # happens to give useful values, but other implementations are not guaranteed
 # to do this.
-#LOCALE_CHARSET:FALSE
+LOCALE_CHARSET:TRUE
 
 
 .h2 ASSUME_CHARSET
@@ -424,7 +424,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # ASSUME_CHARSET can also be changed via the 'o'ptions menu but will
 # not be saved as permanent value in user's .lynxrc file to avoid more chaos.
 #
-#ASSUME_CHARSET:iso-8859-1
+ASSUME_CHARSET:utf-8
 
 
 .h2 ASSUMED_DOC_CHARSET_CHOICE
@@ -552,7 +552,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # the 'o'ptions menu and save that preference in their RC file.
 # This may be a comma-separated list of languages in decreasing preference.
 #
-#PREFERRED_LANGUAGE:en
+#PREFERRED_LANGUAGE:
 
 
 .h2 PREFERRED_CHARSET
@@ -571,7 +571,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # the sending of an unacceptable response is also allowed.  See RFC 2068
 # (http://www.ics.uci.edu/pub/ietf/uri/rfc2068.txt).
 #
-#PREFERRED_CHARSET:
+PREFERRED_CHARSET:utf-8
 
 
 .h2 CHARSETS_DIRECTORY
@@ -652,22 +652,22 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # Set the threshold # of lines Lynx must render before it
 # redraws the screen in PARTIAL mode.  Anything < 0 implies
 # use of the screen size.
-#PARTIAL_THRES:-1
+PARTIAL_THRES:3
 
 
 .h2 SHOW_KB_RATE
 # While getting large files, Lynx shows the approximate rate of transfer.
-# Set this to change the units shown.  "Kilobytes" denotes 1024 bytes:
+# Set this to change the units shown.
 #	NONE to disable the display of transfer rate altogether.
-#	TRUE or KB for Kilobytes/second.
+#	TRUE or KB for Kibibytes/second.
 #	FALSE or BYTES for bytes/second.
-#	KB,ETA to show Kilobytes/second with estimated completion time.
-#	BYTES,ETA to show BYTES/second with estimated completion time.
+#	KB,ETA to show Kibibytes/second with estimated completion time.
+#	BYTES,ETA to show bytes/second with estimated completion time.
 # Note that the "ETA" values are available if USE_READPROGRESS was defined.
-#SHOW_KB_RATE:TRUE
+SHOW_KB_RATE:BYTES,ETA
 
 .h2 SHOW_KB_NAME
-# Set the abbreviation for Kilobytes (1024).
+# Set the abbreviation for Kibibytes (1024 Bytes).
 # Quoting from
 #	http://www.romulus2.com/articles/guides/misc/bitsbytes.shtml
 # In December 1998, the International Electrotechnical Commission (IEC)
@@ -677,9 +677,6 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # metric prefixes and adding the first two letters of the word "binary".  Thus,
 # for instance, instead of Kilobyte (KB) or Gigabyte (GB), the new terms would
 # be kibibyte (KiB) or gibibyte (GiB).
-#
-# If you prefer using the conventional (and more common) "KB", modify this
-# setting.
 #SHOW_KB_NAME:KiB
 
 .h1 Timeouts
@@ -1076,7 +1073,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # default or that defined here can be toggled via the -force_secure
 # command line switch.
 #
-#FORCE_SSL_COOKIES_SECURE:FALSE
+FORCE_SSL_COOKIES_SECURE:TRUE
 
 
 .h1 Internal Behavior
@@ -1263,7 +1260,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # line break.  Note that the valid way to insert extra blank lines in HTML
 # is via a PRE block with only newlines in the block.
 #
-#COLLAPSE_BR_TAGS:TRUE
+COLLAPSE_BR_TAGS:FALSE
 
 
 .h2 TAGSOUP
@@ -1366,7 +1363,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # The default is FALSE, so that the feature needs to be enabled here
 # explicitly if you want it.
 #
-#PERSISTENT_COOKIES:FALSE
+PERSISTENT_COOKIES:TRUE
 
 
 .h2 COOKIE_FILE
@@ -1920,7 +1917,7 @@ DEFAULT_INDEX_FILE:http://lynx.isc.org/
 # prefer the more conservative action of requiring an explicit Y or y to
 # confirm.  The default defined here will override that in userdefs.h.
 #
-#QUIT_DEFAULT_YES:TRUE
+QUIT_DEFAULT_YES:FALSE
 
 
 .h1 HTML Parsing
@@ -3208,7 +3205,7 @@ COLOR:6:brightred:black
 # otherwise. Set this setting to TRUE if you prefer numbered links, but wish
 # to get valid HTML source when printing or mailing when in psrc view.
 # Default is FALSE.
-#PRETTYSRC_VIEW_NO_ANCHOR_NUMBERING:FALSE
+PRETTYSRC_VIEW_NO_ANCHOR_NUMBERING:TRUE
 
 
 .h1 HTML Parsing
@@ -3241,13 +3238,11 @@ COLOR:6:brightred:black
 
 .h2 JUSTIFY
 # JUSTIFY - Appearance
-# This option mirrors command-line option with same name.  Default is TRUE.  If
-# true, most of text (except headers and like this) will be justified.  This
-# has no influence on CJK text rendering.
-#
-# This option is only available if Lynx was compiled with EXP_JUSTIFY_ELTS.
+# This option mirrors command-line option with same name.  Default is FALSE.
+# If true, most of text (except headers and like this) will be justified.
+# This has no influence on CJK text rendering.
 #
-#JUSTIFY:FALSE
+JUSTIFY:TRUE
 
 .h2 JUSTIFY_MAX_VOID_PERCENT
 # JUSTIFY_MAX_VOID_PERCENT - Appearance
@@ -3275,7 +3270,7 @@ COLOR:6:brightred:black
 # explicit activation can also be requested with the -tna command line
 # option.
 #
-#TEXTFIELDS_NEED_ACTIVATION:FALSE
+TEXTFIELDS_NEED_ACTIVATION:TRUE
 
 .h2 LEFTARROW_IN_TEXTFIELD_PROMPT
 # LEFTARROW_IN_TEXTFIELD_PROMPT
@@ -3285,7 +3280,7 @@ COLOR:6:brightred:black
 # only if the contents of the fields have been changed since entering it.
 # If set to TRUE, the confirmation prompt is always issued.
 #
-#LEFTARROW_IN_TEXTFIELD_PROMPT:FALSE
+LEFTARROW_IN_TEXTFIELD_PROMPT:TRUE
 
 
 .h1 Timeouts
@@ -3322,7 +3317,7 @@ COLOR:6:brightred:black
 # one lynx.cfg setting.
 .nf
 #ENABLE_LYNXRC:accept_all_cookies:ON
-#ENABLE_LYNXRC:assume_charset:OFF
+ENABLE_LYNXRC:assume_charset:ON
 #ENABLE_LYNXRC:bookmark_file:ON
 #ENABLE_LYNXRC:case_sensitive_searching:ON
 #ENABLE_LYNXRC:character_set:ON
@@ -3337,34 +3332,35 @@ COLOR:6:brightred:black
 #ENABLE_LYNXRC:emacs_keys:ON
 #ENABLE_LYNXRC:file_editor:ON
 #ENABLE_LYNXRC:file_sorting_method:ON
-#ENABLE_LYNXRC:force_cookie_prompt:OFF
-#ENABLE_LYNXRC:force_ssl_prompt:OFF
-#ENABLE_LYNXRC:ftp_passive:OFF
+ENABLE_LYNXRC:force_cookie_prompt:ON
+ENABLE_LYNXRC:force_ssl_prompt:ON
+ENABLE_LYNXRC:ftp_passive:ON
 #ENABLE_LYNXRC:kblayout:ON
 #ENABLE_LYNXRC:keypad_mode:ON
 #ENABLE_LYNXRC:lineedit_mode:ON
 #ENABLE_LYNXRC:locale_charset:ON
-#ENABLE_LYNXRC:make_links_for_all_images:OFF
-#ENABLE_LYNXRC:make_pseudo_alts_for_inlines:OFF
+ENABLE_LYNXRC:make_links_for_all_images:ON
+ENABLE_LYNXRC:make_pseudo_alts_for_inlines:ON
 #ENABLE_LYNXRC:multi_bookmark:ON
 #ENABLE_LYNXRC:personal_mail_address:ON
 #ENABLE_LYNXRC:preferred_charset:ON
-#ENABLE_LYNXRC:preferred_encoding:OFF
+ENABLE_LYNXRC:preferred_encoding:ON
 #ENABLE_LYNXRC:preferred_language:ON
-#ENABLE_LYNXRC:preferred_media_types:OFF
-#ENABLE_LYNXRC:raw_mode:OFF
+ENABLE_LYNXRC:preferred_media_types:ON
+ENABLE_LYNXRC:raw_mode:ON
 #ENABLE_LYNXRC:run_all_execution_links:ON
 #ENABLE_LYNXRC:run_execution_links_on_local_files:ON
-#ENABLE_LYNXRC:scrollbar:OFF
+ENABLE_LYNXRC:scrollbar:ON
 #ENABLE_LYNXRC:select_popups:ON
-#ENABLE_LYNXRC:set_cookies:OFF
+ENABLE_LYNXRC:set_cookies:ON
 #ENABLE_LYNXRC:show_color:ON
 #ENABLE_LYNXRC:show_cursor:ON
 #ENABLE_LYNXRC:show_dotfiles:ON
-#ENABLE_LYNXRC:show_kb_rate:OFF
+ENABLE_LYNXRC:show_kb_name:OFF
+ENABLE_LYNXRC:show_kb_rate:ON
 #ENABLE_LYNXRC:sub_bookmarks:ON
 #ENABLE_LYNXRC:tagsoup:OFF
-#ENABLE_LYNXRC:underline_links:OFF
+ENABLE_LYNXRC:underline_links:ON
 #ENABLE_LYNXRC:user_mode:ON
 #ENABLE_LYNXRC:useragent:OFF
 #ENABLE_LYNXRC:verbose_images:ON
